Makefile

```diff
@@ -86,8 +86,10 @@ install: all
 	$(MKDIR) $(SHAREDIR)
 	$(CP) script xsd xslt vendor lib.sh $(SHAREDIR)
 	$(MKDIR) $(BINDIR)
-	sed '/^SHAREDIR=/c SHAREDIR="$(SHAREDIR)"' bin/ocr-transform.sh > $(BINDIR)/ocr-transform
-	sed '/^SHAREDIR=/c SHAREDIR="$(SHAREDIR)"' bin/ocr-validate.sh > $(BINDIR)/ocr-validate
+	sed '/^SHAREDIR=/c SHAREDIR="$(SHAREDIR)"' bin/ocr-transform.sh | \
+	  sed "s/[$$](git describe)/$$(git describe)/" > $(BINDIR)/ocr-transform
```

Comment:
This will only work correctly, if the repo was cloned with git. However, it is also possible to download the zip file of a release and install it from there. The second case is easier, as we then only have to use `PKG_VERSION` which is defined here some lines above. Can we make a test whether we install from a git repository or not and then make something different for each case?